### PR TITLE
Reverse direction of rsync in pxf sync command

### PR DESCRIPTION
**Pull request by oliverralbertini**
`pxf sync` is documented as being run from master, i.e. `pxf sync
remote` should sync $PXF_CONF/{conf,lib,servers} from master to remote.
Instead, it syncs from remote to local. This is a vestige of the
original implementation of `pxf cluster sync`, which has now been
changed.

Authored-by: Oliver Albertini <oalbertini@pivotal.io>

### CODE REVIEW

**Comment by oliverralbertini** [comment]
Here's the output from manual testing on an environment with a standby master `smdw`:

```
[gpadmin@mdw ~]$ echo hello > $PXF_CONF/servers/fake-server.xml
[gpadmin@mdw ~]$ for i in smdw sdw{1..3}; do echo -e "$i:"; ssh $i 'cat $PXF_CONF/servers/fake-server.xml'; echo ; done
smdw:
cat: /home/gpadmin/pxf/servers/fake-server.xml: No such file or directory

sdw1:
cat: /home/gpadmin/pxf/servers/fake-server.xml: No such file or directory

sdw2:
cat: /home/gpadmin/pxf/servers/fake-server.xml: No such file or directory

sdw3:
cat: /home/gpadmin/pxf/servers/fake-server.xml: No such file or directory

[gpadmin@mdw ~]$ pxf sync smdw
[gpadmin@mdw ~]$ for i in smdw sdw{1..3}; do echo -e "$i:"; ssh $i 'cat $PXF_CONF/servers/fake-server.xml'; echo ; done
smdw:
hello

sdw1:
cat: /home/gpadmin/pxf/servers/fake-server.xml: No such file or directory

sdw2:
cat: /home/gpadmin/pxf/servers/fake-server.xml: No such file or directory

sdw3:
cat: /home/gpadmin/pxf/servers/fake-server.xml: No such file or directory

[gpadmin@mdw ~]$ pxf cluster sync
Syncing PXF configuration files to 4 hosts...
PXF configs synced successfully on 4 out of 4 hosts
[gpadmin@mdw ~]$ for i in smdw sdw{1..3}; do echo -e "$i:"; ssh $i 'cat $PXF_CONF/servers/fake-server.xml'; echo ; done
smdw:
hello

sdw1:
hello

sdw2:
hello

sdw3:
hello
```

File: concourse/scripts/install_pxf.bash [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -euxo pipefail
 
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 GPHOME="/usr/local/greenplum-db-devel"
-MASTER_HOSTNAME=$( < cluster_env_files/etc_hostfile grep "mdw.*" | awk '{print $2}')
+MASTER_HOSTNAME=$(grep -E '\bmdw.*' cluster_env_files/etc_hostfile | awk '{print $2}')
```
Comment (frankgh) [inline concourse/scripts/install_pxf.bash]: Is this supposed to be part of different PR?

Comment (oliverralbertini) [reply concourse/scripts/install_pxf.bash]: Yeah but it is needed to get the pipeline with a standby master, I will take it out before merging.
